modules/admin-ui/package.json

```diff
@@ -36,11 +36,11 @@
     "karma": "^4.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-coverage": "^1.1.2",
+    "karma-detect-browsers": "^2.0",
     "karma-firefox-launcher": "^1.1.0",
     "karma-jasmine": "^1.1.2",
     "karma-ng-html2js-preprocessor": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "phantomjs-prebuilt": "^2.1.16",
+    "karma-safari-launcher": "^1.0.0",
```

Comment:
Since the plugin list was removed, does this still have to be here?

Reply:
Yes, otherwise safari doesn't work.

This autodetection has a nasty side effect: if it can't find anything then it just sits forever. We need to cast a wide net, otherwise someone with a default Mac install will just be stuck.

Reply:
But this will result in a build error in the end, right? Did I understand that correctly? Because being stuck forever would suck.

Reply:
Yep, line 78 in karma.conf throws an error if nothing is detected.